cmd/cluster/support/post.go

```diff
@@ -125,15 +115,15 @@ func validateResolutionString(res string) error {
 func (p *Post) check() error {
 	if p.Template != "" {
 		if p.Problem != "" || p.Resolution != "" || p.Misconfiguration != "" || p.Evidence != "" {
-			return fmt.Errorf("\nIf Template flag is present, --problem, --resolution, --misconfiguration and --evidence flags cannot be used")
+			return fmt.Errorf("\nIf --template flag is used, --problem, --resolution, --misconfiguration and --evidence flags cannot be used")
 		}
 	} else {
+		if p.Problem == "" || p.Resolution == "" || p.Misconfiguration == "" {
+			return fmt.Errorf("\nIn the absence of --template flag, --problem, --resolution and --misconfiguration flags are mandatory")
+		}
 		if err := validateResolutionString(p.Resolution); err != nil {
```

Comment:
This was the source of the panic, we were trying to slice parse the `Resolution` flag before checking if we needed it. By moving the `if p.Problem == "" || p.Resolution == "" || p.Misconfiguration == ""` up before this, the user gets a much better help output when missing flags.